index.d.ts

```diff
@@ -160,21 +168,20 @@ declare namespace Moleculer {
 
 		waitForServices(serviceNames: string | Array<string>, timeout?: number, interval?: number): Bluebird<void>;
 
-		_init: () => void;
-		_start: () => Bluebird<void>;
-		_stop: () => Bluebird<void>;
-		[name: string]: any;
+		_init(): void;
+		_start(): Bluebird<void>;
+		_stop(): Bluebird<void>;
 	}
```

Comment:
Why did you remove this `[name: string]: any`? The schema contains any other properties, because mixins can handle them.

Reply:
Sorry was looking at the effect of removing it as ES6 classes shouldn't have this type of index definition. It removes much of the type safety of the class. I forgot to put it back in after my testing.